runtime/doc/vim_diff.txt

```diff
@@ -17,7 +17,9 @@ the differences.
 
 - Use `$XDG_CONFIG_HOME/nvim/init.vim` instead of `.vimrc` for configuration.
 - Use `$XDG_CONFIG_HOME/nvim` instead of `.vim` to store configuration files.
-- Use `$XDG_DATA_HOME/nvim/shada/main.shada` instead of `.viminfo` for persistent
+- Use `$XDG_STATE_HOME/nvim` instead of `current working directory` to store
```

Comment:
No need to put "current working directory" in ``.

In fact I'm not sure this line is necessary at all. Vim doesn't place persistent session information in the current working directory (at least, not in any ways that Nvim doesn't also do).